raster/r.out.ascii/r.out.ascii.md

```diff
@@ -1,43 +1,62 @@
 ## DESCRIPTION
 
 *r.out.ascii* converts a user-specified raster map layer
-(**input**=*name*) into an ASCII grid in a text file (**output**=*name*)
+(**input=***name*) into an ASCII grid in a text file (**output=***name*)
```

Comment:
@petrasovaa or @cwhite911 What is the convention used throughout the other pages now?